set_vcf_sample_id/README.md

```diff
@@ -0,0 +1,185 @@
+# Update a VCF header line sample ID
+
+This pipeline is useful when you have a set of single-sample VCFs
+in Cloud Storage in which the sample ID in the header needs to be changed.
```

Comment:
Agreed that Dataflow has full integration with the Genomics API, but integrating Dataflow with any language is fairly straightforward - I'm not seeing that as a bottleneck.  Yes it is Java, but you can modularize that to make it callable from other languages.

Regarding accessing the Genomics API from the Pipeline API, I'm not seeing that connection quite yet.  Do you mean that you can run a `cmd` to get data from the Genomics API, or that you have a direct integration via the Pipeline API?  Do you have any examples? 

I'm not against having the ability of the Pipeline API to provide traditional file-based data access, but there is no experimental and sample annotation of the data in the Genomics API of these files, even if these files are not loaded in the Genomics API.  Basically what will happen, is that people will begin using the Pipeline API more and more with time since it is so convenient, and not bother with the Genomics API.  Any annotation regarding the VCFs or BAMs is non-existent since they are just files, so there is no way to perform correlations among datasets.  If these files get removed, then the provenance is also gone.  And building a query system using relational algebra or even via something like PyTables is not that difficult in the cloud.  Even using something simple by implementing Sorted Strings Table (SSTable) files would allow users to bypass BigQuery.

Basically the point I am driving is that you have a nice beginning built with all these tools, especially the Genomics API.  Processing at large scale is just the beginning, as this has been done many times before.  What has not been done before is to have large scale modular integration with annotation of all types.  So there just a little more work to do, by adding the extra functionality to provide tight integration with a complete turn-key solution, or otherwise you're throwing the baby out with the bathwater, and then users keep shifting their data to the "next great" cloud-enabled data-processing platform.  Basically one then loses the reliable and expected consistency to build a fruitful user-community from.